src/date.rs

```diff
@@ -10,6 +10,8 @@ use core::ops::{Add, Sub};
 use core::{fmt, hash};
 use oldtime::Duration as OldDuration;
 
+#[cfg(any(all(feature = "locales", any(feature = "alloc", feature = "std")), test))]
+use format::DelayedFormatLocalized;
```

Comment:
This should just be `#[cfg(feature = "locales")]`, anything else probably suggests that something is awry.

Reply:
I copied DelayFormatLocalized from DelayFormat which has all of this. I just added the condition.

https://github.com/chronotope/chrono/blob/febb8dc168325ac471b54591c925c48b6a485962/src/format/mod.rs#L683-L694

I can put it only on locales if you think it's correct (I'm really not sure)

Reply:
I believe this is really necessary as the feature locales won't work without alloc or std. If we reduce to only `feature = "locales"` it means that building with `--no-default-features --features locales` will fail with non really descriptive error messages